pytest_reportportal/plugin.py

```diff
@@ -78,6 +79,23 @@ def pytest_configure_node(node):
                                                       py_test_service)
 
 
+def is_portal_on_maintenance(session):
```

Comment:
You should process the exception within service initialization instead of writing a code that has already been written.

Reply:
Removed function, added exception handler.